docs/Building.md

```diff
@@ -4,47 +4,66 @@
+    
+1. *(Skip if you are not interested in running the tests.)*\
+ Install FDB on your development machine. The latest FDB binaries can be found on
+the [FoundationDB website](https://www.foundationdb.org/download/).
 
-The Record Layer project does not use the Gradle IDEA plugin, unlike many similar projects. Instead, it uses IntelliJ IDEA's builtin Gradle support to compile the project and run the tests; the repository will run a "sync" with the Gradle configuration every time the project is opened or a file referenced from `build.gradle` is changed.
+1. Install and launch [IntelliJ IDEA](https://www.jetbrains.com/idea/). This document is based on Version 2020.2.3.
 
-If you are used to opening older `.ipr`-based IntelliJ projects, the process for opening this project will be somewhat different. Instead of selecting the `fdb-record-layer.ipr` file, open the project's root directory; IntelliJ will find the project configuration in the `.idea` directory.
+1. In the "Welcome to IntelliJ IDEA" window, click "Open or import" and open the project's root directory. (If you do not see the welcome window, File > Open can do the same.)
 
-If you make changes to most IntelliJ settings, they will save over the files that are checked into Git. These changes can be shared by committing the updated files.
+1. In the Gradle tool window (expand by clicking the Gradle button on right top), click ![the Reload All Gradle Projects button](https://resources.jetbrains.com/help/img/idea/2020.2/icons.actions.refresh.svg) on the toolbar.
 
-There are a few settings that we use for consistency that are saved in the user-specific `workspace.xml` file. These settings must be enabled manually by each developer:
+1. Build > Build Project.
 
-* Preferences > Editor > General > Auto Import > Optimize imports on the fly (for current project)
+1. Preferences > Editor > General > Auto Import > Optimize imports on the fly (for current project)
 
-* (optional, but useful) Preferences > Build, Execution, and Deployment > Build Tools > Maven > Importing > Automatically download > Sources
+1. *(This is optional, but useful.)*\
+ Preferences > Build, Execution, and Deployment > Build Tools > Maven > Importing > Automatically download > Sources
 
-Before attempting to build anything, exit IntelliJ and build once from the command line. This will generate the protobuf files.
+**Now you have configured IntelliJ successfully!**
 
-```
-./gradlew package
-```
+*(Skip if you are not interested in running the tests.)*\
+To verify that you have a working configuration, pick some tests that depend on FDB (such as those in the `FDBRecordStoreQueryTest` fixture), and check you can run them from IntelliJ.
 
-Launch IntelliJ again and Build > Build Project should succeed.
+If this does not work smoothly for you, please create a [new issue](https://github.com/FoundationDB/fdb-record-layer/issues/new) or ask in [the forum](https://forums.foundationdb.org/c/using-layers).
```

Comment:
I am surprisingly ignorant of how the FDB cluster is actually found by the tests/record layer.  Maybe it is something that anyone familiar with FDB would know, but I thought I'll ask:  Should we include directions on either environment variables that should be set or about the `feb-environment.properties` file?

Reply:
Yes. The directions at the end of this doc. I think the reason that it is at the end is that users who use the default settings of FDB should not need set `fdb-environment.properties` at all, so they shouldn't be confused about it.